app/src/main/AndroidManifest.xml

```diff
@@ -73,6 +73,7 @@
       android:name=".app.profile.AddProfileActivity"
       android:theme="@style/OppiaThemeNoActionBarColorAccentColorPrimary" />
     <activity
+      android:label="@string/admin_auth_controls_label"
       android:name=".app.profile.AdminAuthActivity"
       android:theme="@style/OppiaThemeWithoutActionBar"
```

Comment:
`android:label="@string/admin_auth_controls_label"` This should come below `theme` attribute